velox/connectors/hive/storage_adapters/gcs/GCSFileSystem.cpp

```diff
@@ -175,6 +180,7 @@ class GCSReadFile final : public ReadFile {
   }
 
   std::shared_ptr<gcs::Client> client_;
+  std::function<void(std::shared_ptr<gcs::Client>&, const std::optional<std::string>&, const std::optional<std::vector<std::string>>&)> refreshTokenFn_;
```

Comment:
Please add a comment here about the expected inputs are for the `refereshTokenFn_`